#
# Definition of all input parameters for the filter
#
input_cloud:                 '/kinect2/sd/points'
output_cloud:                '/scene/table'
camera_frame_name:           '/kinect2_rgb_optical_frame'
x_min:                        -0.4
x_max:                        0.31 
y_min:                        -0.43
y_max:                        0.25
z_min:                        0.6
z_max:                        1.3
stat_mean:                    10
stat_std:                     0.5
pcl_viz:                      false